app/scripts/lib/createDupeReqFilterStream.ts

```diff
@@ -54,7 +54,8 @@ export default function createDupeReqFilterStream() {
     transform(chunk: JsonRpcRequest, _, cb) {
       // JSON-RPC notifications have no ids; our only recourse is to let them through.
       const hasNoId = chunk.id === undefined;
-      const requestNotYetSeen = seenRequestIds.add(chunk.id);
+      const requestNotYetSeen =
+        chunk.id !== null && seenRequestIds.add(chunk.id);
```

Comment:
I'm a bit concerned that this might have a functional impact, for requests with an id of `null`. I don't know why we'd get requests like that, but, I wasn't able to rule it out. I think we can allow null as a seen request ID, rather than adding a condition for it here.

Reply:
- Reverted this change: fe7c6ad17978cf7d8f60d25e46f6ccffe1b1f7d4
- Created a PR for reverting this in `develop`: https://github.com/MetaMask/metamask-extension/pull/26814